### PR TITLE
Remove Pivotal Tracker

### PR DESCRIPTION
**Pull request by eminogrande**
deprecated

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ Comes with a naming system, for developerability.
 Written in Erlang.
 
 To install and run the Aeternity node, see the instructions [below](#how-to-start)
-or just [follow the progress of the project in Pivotal Tracker][pivotal].
+or just follow the progress of the project via GitHub Issues ([Deprecated Project Management: Pivotal Tracker][pivotal]).
```
Comment (tolbrino) [inline README.md]: Its not just deprecated, but simply not supported anymore. So it shouldn't be linked here.

Comment (eminogrande) [reply README.md]: Great! Thank you Tino :)

**Comment by eminogrande** [comment]
Resolved. Thank you for the feedback.

File: README.md [hunk 2]
```diff
@@ -14,11 +14,9 @@ Comes with a naming system, for developerability.
 
 Written in Erlang.
 
-To install and run the Aeternity node, see the instructions [below](#how-to-start)
-or just [follow the progress of the project in Pivotal Tracker][pivotal].
+To install and run the Aeternity node, see the instructions [below](#how-to-start) or just follow the progress of the project via GitHub Issues.
 
-If you are interested in participating in a security bounty, check our
-[HackerOne Aeternity Bounty Program][hackerone].
+If you have discovered a bug or security vulnerability please get in touch. The aeternity Crypto Foundation pays bug-bounties up to 100.000 AE Tokens for critical vulnerabilities. Please get in touch via security@aeternity-foundation.org
```
Comment (tolbrino) [inline README.md]:
```suggestion
If you have discovered a bug or security vulnerability please get in touch. The aeternity Crypto Foundation pays bug-bounties up to 100.000 AE Tokens for critical vulnerabilities. Please get in touch via [security@aeternity-foundation.org](mailto:security@aeternity-foundation.org).
```

**Pull request merged**